src/construct.yaml

```diff
@@ -0,0 +1,25 @@
+{% from "version.jinja" import version %}
+{% from "python.jinja" import py_ver_maj, py_ver_min %}
+
+
+name: miniforge-py{{ py_ver_maj }}{{ py_ver_min }}
+version: {{ version }}
+
+install_in_dependency_order: true
```

Comment:
Require dependency order is satisfied. Should also solve dependencies while we are at it. Also should make sure any post-link scripts can execute knowing their dependencies are present and properly installed.